src/client/auth-service.js

```diff
@@ -3,27 +3,63 @@ import auth0 from "auth0-js";
 const baseURL =
   window.BASE_URL || `${window.location.protocol}//${window.location.host}`;
 
+const loginStrategies = {
```

Comment:
A good first step -- I think the next one, we'd want to move all this into src/integrations/auth/ files/directories and this could be in the react-component.js file.